data/loginPage.ts

```diff
@@ -0,0 +1,37 @@
+/* Assign Details to the banner*/
```

Comment:
You don't have to export each and every variable like this instead try a structure like below.

export const loginPageData = {
  loginPageTitle: "",
  loginPageDescription: ""
  loginButtonCaption: 
  loginEmailtxtCaption: 
};